internal/config/config.go

```diff
@@ -93,6 +95,10 @@ func (s *Config) Load() error {
 
 	v.Set("config", v.ConfigFileUsed())
 
+	if s.GlobalFlags.Debug {
+		debug.Debug("Load cli", "User config after loading and setting up service")
+		v.Debug()
```

Comment:
Changed this since the `viper.debug()` is a dump of viper config, not really having anything to do with the `log.debug()` also flume handles debug level, so it should now be better overall. 

current:
```
	log.Debug("Config: Load()", "Viper config logs", v.AllSettings())
```